tethys_compute/models/condor/condor_base.py

```diff
@@ -111,12 +116,31 @@ def _log_files(self):
         """
         pass
 
-    @abstractmethod
-    def _get_logs_from_remote(self, log_files, ):
+    def _get_logs_from_remote(self, log_files):
         """
-        Define in condor_workflow. Since we need the instance of condorpy_workflow.
+        Get logs from remote while job is running.
+
+        Args:
+            log_files: the nested dictionaries of all the log files
+
+        Returns: the contents of all the logs files in a nested dictionary.
         """
-        pass
+
+        contents = dict()
+        for log_file_type, log_file_path in log_files.items():
+            if log_file_type == 'workflow':
+                contents['workflow'] = partial(self.get_log_content, log_file_path)
+            else:
+                # Parse out logs for each job.
+                contents[log_file_type] = dict()
+                for job_log_type, job_log_path in log_file_path.items():
+                    contents[log_file_type][job_log_type] = partial(self.get_log_content, job_log_path)
+
+        return contents
+
+    def get_log_content(self, log_file):
+        content, _ = self.condor_object.execute(['cat', log_file])
```

Comment:
```suggestion
        content, _ = self.condor_object._execute(['cat', log_file])
```

Reply:
If only I were a little more familiar with that `condorpy` library. Then I wouldn't make little mistakes like this.